bin/phpunit-sqlite-wrapper

```diff
@@ -0,0 +1,36 @@
+<?php
+if (!isset($argv[1])) {
+    fwrite(STDERR, 'First parameter for sqlite database file required.');
+    exit(1);
+}
+
+$db = new PDO('sqlite:' . $argv[1]);
+
+// git working copy
+if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
+    require_once __DIR__ . '/../vendor/autoload.php';
+}
+// Composer installation
+if (file_exists(__DIR__ . '/../../../autoload.php')) {
+    require_once __DIR__ . '/../../../autoload.php';
+}
+
+while ($test = $db->query('SELECT id, command FROM tests WHERE reserved_by_process_id IS NULL ORDER BY file_name LIMIT 1')->fetch()) {
+    $db->prepare('UPDATE tests SET reserved_by_process_id = :procId WHERE id = :id')
```

Comment:
To prevent race conditions, after running this `UPDATE` query you should check the affected row count result. If affected rows is `0`, that means another process has already successfully run the UPDATE query for the same test, and you should do a `continue;` to try grabbing another set with `SELECT`.

Reply:
Oh, and also in the UPDATE query you should have `WHERE id = :id AND reserved_by_process_id IS NULL` in case another process already updated that column.